tests/integration/test_schema.py

```diff
@@ -2674,6 +2674,12 @@ async def test_schema_hard_delete_whole_schema(registry_async_client: Client) ->
     assert res.json()["error_code"] == 40401
     assert res.json()["message"] == f"Subject '{subject}' not found."
 
+    # Check that fetching unescaped schema gives valid error message
```

Comment:
question: Just looking at this test, it's quite hard to see why escaping is relevant here. Isn't the reason for 404 that the schema was deleted?

Reply:
There are two endpoints to fetch schema, `/subjects/{{subject}}/versions/{{version}}` returning JSON object and `/subjects/{{subject}}/versions/{{version}}/schema` returning only "unescaped" or "raw" schema. The first endpoint was handling error already fine & tests, the second one was somehow missing this exception handling

Reply:
Ah, that makes sense, thanks for explaining 🙏